### PR TITLE
[Ru-Translation] More on staking

### PR DESCRIPTION
**Pull request by Serhioromano**
This time some spelling fixed. Also in some places stake was translated as bid and some very confusing terminologies was invented.

### CODE REVIEW

File: public/locales/ru-RU.json [hunk 1]
```diff
@@ -380,7 +380,7 @@
   "Opening Block": "Открывающий блок",
   "Closing Block": "Закрывающий блок",
   "Total Value Locked (TVL)": "Общая заблокированная стоимость (TVL)",
-  "Automatic restaking": "Автоматический повторный стейкинг",
```
Comment (Serhioromano) [inline public/locales/ru-RU.json:383]: This term was invented and is not very intuitive. I changed it on someting any one can understand without looking for explanation

File: public/locales/ru-RU.json [hunk 2]
```diff
@@ -534,8 +534,8 @@
   "%symbol% required": "Необходимо %symbol%",
   "Your %symbol% earnings have also been harvested to your wallet!": "Ваши поступления %symbol% также собраны в ваш кошелек!",
   "Your %symbol% funds have been staked in the pool!": "Ваши средства %symbol% вложены в пул!",
-  "Max stake for this pool: %amount% %token%": "Максимальная ставка в этом пуле: %amount% %token%",
-  "Maximum total stake: %amount% %token%": "Общая максимальная ставка: %amount% %token%",
+  "Max stake for this pool: %amount% %token%": "Максимальный вклад в этом пуле: %amount% %token%",
+  "Maximum total stake: %amount% %token%": "Общая максимальный вклад: %amount% %token%",
```
Comment (ChefNina) [inline public/locales/ru-RU.json:538]: ОбщИЙ максимальный вклад 👀

Comment (Serhioromano) [reply public/locales/ru-RU.json:538]: Поправляю

File: public/locales/ru-RU.json [hunk 3]
```diff
@@ -534,8 +534,8 @@
   "%symbol% required": "Необходимо %symbol%",
   "Your %symbol% earnings have also been harvested to your wallet!": "Ваши поступления %symbol% также собраны в ваш кошелек!",
   "Your %symbol% funds have been staked in the pool!": "Ваши средства %symbol% вложены в пул!",
-  "Max stake for this pool: %amount% %token%": "Максимальная ставка в этом пуле: %amount% %token%",
-  "Maximum total stake: %amount% %token%": "Общая максимальная ставка: %amount% %token%",
+  "Max stake for this pool: %amount% %token%": "Максимальный вклад в этом пуле: %amount% %token%",
+  "Maximum total stake: %amount% %token%": "Общий максимальный вклад: %amount% %token%",
```
Comment (Serhioromano) [inline public/locales/ru-RU.json:538]: Исправил

**Review by Chef-Cheems — APPROVED** [review]

File: public/locales/ru-RU.json (hunk 3 above)
Comment (ChefNina) [reply public/locales/ru-RU.json:538]: ✔️

**Pull request merged**